### PR TITLE
Updated headers

### PR DESCRIPTION
**Pull request by lelandg559**

### CODE REVIEW

**Comment by lelandg559** [comment]
I plan to merge common code from header and largeheader together but I am not ready to do that yet.

**Comment by jfhamlin** [comment]
> I plan to merge common code from header and largeheader together but I am not ready to do that yet.

👍

File: app/components/Header.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
 
 import React from 'react';
 import { Link } from 'react-router';
+let bgImage = require('../assets/images/bg.png');
+let smallLogo = require('../assets/images/logo-small-white@3x.png');
```
Comment (jfhamlin) [inline app/components/Header.js]: Might be cool to throw a js module somewhere, maybe under `/assets`, that provides an index of all images so we don't have this boilerplate.

Comment (lelandg559) [inline app/components/Header.js]: yeap, ill review that when I make a second pass for internal cleanup.

**Comment by lelandg559** [comment]
Sondra gave LGTM

**Pull request merged**